### DISCUSSION TITLE
3.10.0

### DISCUSSION THREAD

**Discussion by zenangst** [zenangst/KeyboardCowboy#337] [Announcements]
## Oh, you thought we were done? Nope! 

- ⚡️ We've added support for instant mode when using the "type" command. Now you can type away with lightning speed. #334 
- ⏲️ And guess what? We've also implemented the ability to delay keyboard shortcut execution. Because sometimes, you just need a little extra time to make sure you're hitting all the right keys. #332 
- 💅🏻 But wait, there's more! We've made several minor UI improvements. Because who doesn't love a sleeker and sexier interface?

I would like to give a shout-out to the amazing @Fischmuetze and @rakalex for their absolutely brilliant ideas and feature requests.

Kudos to you both! Keep those amazing ideas coming!

<hr /><em>This discussion was created from the release <a href='https://github.com/zenangst/KeyboardCowboy/releases/tag/3.10.0'>3.10.0</a>.</em>

**Comment by Fischmuetze** [comment]
Great job! THX! 🤩
